### PR TITLE
Storage platform versions update for 1.12

### PR DESCRIPTION
**Pull request by santhoshatdell**
# Description
Adding the Storage platform versions qualified for CSM v1.12

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1435 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?

### CODE REVIEW

File: content/docs/prerequisites/_index.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ weight: 1
...
 | ----------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------: |
-| PowerMax    | PowerMax 2500/8500 PowerMaxOS 10 (6079)<br>PowerMaxOS 10.0.1 (6079)<br>PowerMaxOS 10.1 (6079)<br>PowerMax 2000/8000 - 5978.711.711, 5978.714.714<br>5978.479.479<br>Unisphere 10.0,10.0.1,10.1 |       iscsi-initiator-utils<br>multipathd or powerpath<br>nvme-cli<br>nfs-utils        |
+| PowerMax    | PowerMax 2500/8500 PowerMaxOS 10 (6079)<br>PowerMaxOS 10.0.1 (6079)<br>PowerMaxOS 10.1 (6079)<br> PowerMaxOS 10.2.0 (6079)<br>PowerMax 2000/8000 - 5978.711.711, 5978.714.714<br>5978.479.479<br>Unisphere 10.0,10.0.1,10.1,10.2 |       iscsi-initiator-utils<br>multipathd or powerpath<br>nvme-cli<br>nfs-utils        |
```
Comment (santhoshatdell) [inline content/docs/prerequisites/_index.md:13]: 10.2.0 format is not consistent with 10.1

**Review by donatwork — APPROVED** [review]

**Review by anandrajak1 — APPROVED** [review]

File: content/docs/prerequisites/_index.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ weight: 1
...
-| PowerStore  |                                                                                       3.0, 3.2, 3.5, 3.6                                                                                       |              iscsi-initiator-utils<br>multipathd<br>nvme-cli<br>nfs-utils              |
+| PowerScale  |                                                                                OneFS 9.4, 9.5.0.x (x >= 5), 9.7, 9.8, 9.9                                                                                |                                       nfs-utils                                        |
+| PowerStore  |                                                                                       3.0, 3.2, 3.5, 3.6, 4.0                                                                                       |              iscsi-initiator-utils<br>multipathd<br>nvme-cli<br>nfs-utils              |
```
Comment (sharmilarama) [inline content/docs/prerequisites/_index.md:17]: Please remove 3.0, 3.2 as indicated in EOL roadmap.

File: content/docs/resiliency/_index.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ CSM for Resiliency provides the following capabilities:
 {{<table "table table-striped table-bordered table-sm">}}
 |               | PowerFlex    | Unity XT                          | PowerScale                              | PowerStore                    | PowerMax |
 | ------------- | :----------: | :-------------------------------: | :-------------------------------------: | :---------------------------: | :---------------------------: |
-| Storage Array | 3.6.x, 4.5.x, 4.6.x | 5.2.x, 5.3.0, 5.4.x | OneFS 9.4, 9.5.0.x (x >= 5), 9.7 | 3.0, 3.2, 3.5, 3.6 | 2500/8500 PowerMax OS 10 (6079), Unisphere 10.x |
+| Storage Array | 3.6.x, 4.5.x, 4.6.x | 5.2.x, 5.3.x, 5.4.x | OneFS 9.4, 9.5.0.x (x >= 5), 9.7, 9.8, 9.9 | 3.0, 3.2, 3.5, 3.6, 4.0 | 2500/8500 PowerMax OS 10 (6079), Unisphere 10.x |
```
Comment (sharmilarama) [inline content/docs/resiliency/_index.md:53]: Remove PowerStore 3.0, 3.2

**Review by sharmilarama — DISMISSED** [review]

**Review by mjsdell — APPROVED** [review]
lgtm

**Pull request merged**